src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Since it's important to explicity tell the screen readers about the meaning of this image, I would use an img tag. So I can tell at the 'alt' property it is my logo. 
https://github.com/mate-academy/layout_catalog-test_task